bayes_opt/bayesian_optimization.py

```diff
@@ -347,29 +339,20 @@ def maximize(self,
         self.dispatch(Events.OPTIMIZATION_START)
         self._prime_queue(init_points)
 
-        old_params_used = any([param is not None for param in [acq, kappa, kappa_decay, kappa_decay_delay, xi]])
+        old_params_used = any([param is not None for param in [kappa, kappa_decay, kappa_decay_delay, xi]])
```

Comment:
Maybe this would be a good opportunity to drop these parameters from the call signature.

Reply:
do it! :-)

Reply:
done.